.github/workflows/build-and-deploy.yml

```diff
@@ -5,10 +5,11 @@ name: build-and-deploy
 on:
   push:
     branches:
-      - main
+      - ad/make-bucket-info-secret
```

Comment:
guessing this needs to be switched back after testing
```suggestion
      - main
```

Reply:
Yup thanks forgot to switch it back after testing.